.github/workflows/release.yml

```diff
@@ -28,13 +28,12 @@ jobs:
         run: |
           pushd target/release
           rm suimu-*.d
-          mv suimu-check suimu-check-${{ runner.os }}
-          mv suimu-build suimu-build-${{ runner.os }}
+          mv suimu-check suimu-${{ runner.os }}
```

Comment:
Maybe we shall change `suimu-check` to `suimu`?

Reply:
Ah yes